app/src/main/java/dev/blunch/blunch/utils/Utils.java

```diff
@@ -0,0 +1,15 @@
+package dev.blunch.blunch.utils;
+
+import com.firebase.client.Firebase;
+
+/**
+ * Utils class
+ * @author albert
+ */
+public class Utils {
+
+    public static String generateId() {
+        return new Firebase("https://blunch.firebaseio.com/").push().getKey();
```

Comment:
The future is this afternoon! And we will use it! I will explain tomorrow dont worry.